javascript/app.js

```diff
@@ -0,0 +1,336 @@
+const kristynsClosetLaundry = [
+  "left shoe",
+  "cowboy boots",
+  "right sock",
+  "GA hoodie",
+  "green pants",
+  "yellow knit hat",
+  "marshmallow peeps"
+];
+
+for (let i = 0; i < kristynsClosetLaundry.length; i++) {
+	console.log(`WHIRR: Now washing ${kristynsClosetLaundry[i]}.`);
+}
+
+// Inventory
+// 2. Thom wants to do inventory on his closet. Using bracket notation, log the arrays (not the elements in the arrays) containing all of Thom's shirts, pants, and accessories.
+
+// Not sure I understand the question. If you don't log what's inside, what are you logging?
+for (let i = 0; i < thomsCloset.length; i++) {
+	console.log(`Thom's Inventory: ${thomsCloset[i]}.`);
+}
+
+
+// Multiples of 3 and 5
+// Yes, you might have tackled this earlier, but try it again (don't look back at your other code 👀)
+// If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23.
+// Find the sum of all the multiples of 3 or 5 below 1000.
+let sum = 0;
+for (i = 0; i < 1000; i++) {
+	if (i % 5 === 0 || i % 3 === 0){
+		sum += i;
+	}
+}
+console.log(`The sum of the mutliples is ${sum}.`);
+
+
+// HUNGRY FOR MORE?
+// Triangles
+// declare a variable argument and set it equal to 7.
+// Write a loop that console logs a "left isosceles" triangle (SEE BELOW) made of '#' that has the height and length of argument.
+let argument1 = 7;
+const leftIsosceles = (argument1) => {
+	for (var i = 1; i <= argument1; i++) {
+		console.log('#'.repeat(i));	
+	}
+}
+// call function with an argument which will be inputted in the function
+leftIsosceles(argument1);
+
+// Write a loop that console logs a "right isosceles" triangle (SEE BELOW) made of '#' that has the height and length of argument. This is deceptively tricky.
+let argument2 = 7;
+const rightIsosceles = (argument2) => {
+	for (let i = (argument2 - 1); i >= 0; i--) {
+		console.log(' '.repeat(i) + '#'.repeat(argument2 - i));
+	}
+}
+// call function with an argument which will be inputted in the function
+rightIsosceles(argument2);
+
+// Write a loop that console logs an "upside down left" isosceles triangle made of '#' that has the height and length of the argument.
+let argument3 = 7;
+const upsideDownLeft = (argument3) => {
+	for (var i = argument3; i > 0; i--) {
+		console.log('#'.repeat(i));	
+	}
+}
+// call function with an argument which will be inputted in the function
+upsideDownLeft(argument3);
+
+
+// Write a loop that console logs an "upside down right" isosceles triangle made of '#' that has the height and length of the argument. This is also tricky.
+let argument4 = 7;
+const upsideDownRight = (argument4) => {
+	for (let i = argument4; i > 0; i--) {
+		console.log(' '.repeat(argument4 - i) + '#'.repeat(i));
+	}
+}
+// call function with an argument which will be inputted in the function
+upsideDownRight(argument4);
+
```

Comment:
Great job with this Charlotte!  I did not even know about `.repeat()`.  i love it when i learn about new things from students!